__tests__/Employee.test.js

```diff
@@ -0,0 +1,25 @@
+const Employee = require("../src/lib/Employee");
+
+let bob = new Employee("bob", 1, "bob@gmail.com");
```

Comment:
This variable doesn't change so we should have `const` not `let`. Equally , this is currently in global memory which isn't necessary. You should declare this inside the `describe` block.